aidants_connect_web/templates/public_website/habilitation.html

```diff
@@ -75,7 +71,44 @@ <h2>Liste des structures éligibles à l’utilisation d’Aidants Connect</h2>
             Exemples : les travailleurs sociaux.
           </li>
         </ul>
-        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles.</strong></p>
+        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles. Pour plus d'informations, 
+          nous vous invitons à consulter <a href="https://aidantsconnect.beta.gouv.fr/faq/mandat/">notre FAQ.</a></strong></p><br>
+        
+        <h2>Calendrier du déploiement</h2>
+        
+        <p><strong>Le déploiement de l'outil va s'effectuer par vagues territoriales progressives selon le calendrier suivant :</p><strong>
```

Comment:
Il faut fermer en premier ce que l'on a ouvert en dernier : 

```suggestion
        <p><strong>Le déploiement de l'outil va s'effectuer par vagues territoriales progressives selon le calendrier suivant :</strong></p>
```